docs/3.0rc/develop/global-concurrency-limits.mdx

```diff
@@ -45,39 +48,35 @@ Global concurrency limits can be in an `active` or `inactive` state:
-specific concurrency limit. This is the rate at which these slots "decay" or refresh.
+When you set up a concurrency limit with slot decay:
+1. Each time a slot is occupied, it becomes unavailable for other tasks to use.
+2. The slot eventually becomes available again over time, based on the slot decay rate (i.e. `slot_decay_per_second`).
+3. This creates a "rate limiting" effect, limiting how often slots can be used.
 
-To configure slot decay, set the `slot_decay_per_second` parameter when defining or adjusting a concurrency limit.
+To configure slot decay, set the `slot_decay_per_second` parameter when creating or updating a concurrency limit. This value determines how quickly slots refresh:
 
-For practical use, consider the following:
+- A higher value (e.g., 5.0) means slots refresh quickly. Tasks can run more frequently, but with short pauses between them.
+- A lower value (e.g., 0.1) means slots refresh slowly. Tasks run less frequently, with longer pauses between them.
 
-- *Higher values*: Setting `slot_decay_per_second` to a higher value, such as 5.0, results in slots becoming available relatively quickly. 
-In this scenario, a slot that was occupied by a task will free up after just `0.2` (`1.0 / 5.0`) seconds.
-- *Lower values*: Conversely, setting `slot_decay_per_second` to a lower value, like 0.1, causes slots to become available more slowly. 
-In this scenario it takes `10` (`1.0 / 0.1`) seconds for a slot to become available again after occupancy.
+For example:
 
-Slot decay provides fine-grained control over the availability of slots, enabling you to optimize the rate of your workflow based on your 
-specific requirements.
+- With a decay rate of 5.0, you could run a task roughly every 0.2 seconds.
+- With a decay rate of 0.1, you'd wait about 10 seconds between task runs.
+
+Choose a decay rate that balances your need for frequent task execution with the desire to limit overall system load. This allows you to fine-tune your workflow's performance and resource usage.
 
 ### Through the UI
 
-You can create, read, edit, and delete concurrency limits through the Prefect UI.
+You can manage global concurrency limits in the **Concurrency** section of the Prefect UI.
 
-When creating a concurrency limit, you can specify the following parameters:
+![Concurrency limits in the UI](/images/gcl-1.png)
 
-- **Name**: The name of the concurrency limit. This name is also how to reference the concurrency limit in your code. Special characters, 
-such as `/`, `%`, `&`, `>`, `<`, are not allowed.
-- **Concurrency Limit**: The maximum number of slots that can be occupied on this concurrency limit.
-- **Slot Decay Per Second**: Controls the rate at which slots are released when the concurrency limit is used as a rate limit. You must configure this 
-value when using the `rate_limit` function.
-- **Active**: Whether or not the concurrency limit is in an active state.
```

Comment:
these params are already explained above, we instead replace this with an image of where you configure in the UI

![image](https://github.com/user-attachments/assets/262663b0-c8f2-4258-b557-39c61102ac70)